dubbo-native/src/main/java/org/apache/dubbo/common/serialize/Serialization$Adaptive.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.common.serialize;
+import org.apache.dubbo.common.extension.ExtensionLoader;
+public class Serialization$Adaptive implements org.apache.dubbo.common.serialize.Serialization {
```

Comment:
It would be more useful if `XXX$Adaptive.java` could be generated automatically through a plugin or command line.

Reply:
Plug-in is being developed